controllers/topology/template_cleanup_helpers.go

```diff
@@ -0,0 +1,131 @@
+package topology
+
+import (
```

Comment:
Not really happy about putting those helpers just into a file. Maybe it makes sense to move them into a separate package below internal/ ?

Reply:
+1 moving them under an internal package

Reply:
Done